### PR TITLE
Add Strip Metadata option to Optimize Images

### PR DESCRIPTION
**Pull request by sjaks**
Implement a checkbox so that user can enable metadata stripping
when uploaded images are optimized.

Image optimization is run only for the thumbnails since this hooks to the `image_make_intermediate_size` filter.
The reasoning is that the original images would get backed up. This can be desired for metadata too but for some metadata types the original image probably should be stripped as well. Does @ottok have opinion on this? Should this be extended more extensively?

### CODE REVIEW

**Comment by ottok** [comment]
This looks fine. Also the project template @Seravo/WordPress needs to be updated to set metadata stripping 'on' by default on new sites.

**Comment by sjaks** [comment]
Related: https://github.com/Seravo/wordpress/pull/151

**Review by ottok — APPROVED** [review]
LGTM!

File: modules/sitestatus.php [hunk 1]
```diff
@@ -126,6 +127,13 @@ public static function register_optimize_image_settings() {
         'optimize_images_settings',
         'seravo-optimize-images-settings'
       );
+      add_settings_field(
+        'seravo-strip-image-metadata-field',
+        __('Strip Image Metadata', 'seravo'),
```
Comment (elguitar) [inline modules/sitestatus.php:132]: This is not translated yet. Do you need help with the translations?

**Review by elguitar — COMMENTED** [review]
Looks good. There's one untranslated string.

File: modules/sitestatus.php (hunk 1 above)
Comment (sjaks) [reply modules/sitestatus.php:132]: There were two untranslated strings -> translated.

Comment (sjaks) [reply modules/sitestatus.php:132]: There seems to be some abnormal changes in the translation files - at least to my eye.

Comment (elguitar) [reply modules/sitestatus.php:132]: It seems to revert changes in https://github.com/Seravo/seravo-plugin/commit/617019cb49920ca6be0fffad2e20f7df67a46cdf#diff-d216ba5c5cb4f488966fc057f833dd86133eaee6e3eed47a19db4ce8f73677b5L708-L709

**Comment by sjaks** [comment]
Added translations
image

**Review by elguitar — APPROVED** [review]
LGTM!

**Pull request merged**